### PR TITLE
Feat: Issue Templates to Issue Forms 

### PR DESCRIPTION
**Pull request by 2002Bishwajeet**
Fixes issue #269

### CODE REVIEW

**Comment by lohanidamodar** [comment]
@2002Bishwajeet  Great work. One more change before we can approve. We have made few changes in the language and also the format of Code of conduct. Please check https://github.com/appwrite/appwrite template and update accordingly.
Thank you.

File: .github/ISSUE_TEMPLATE/bug.yaml
Comment (lohanidamodar) [inline .github/ISSUE_TEMPLATE/bug.yaml:37]: Let's add the SDK Generator version and the respective releases on the options instead.

**Review by lohanidamodar — COMMENTED** [review]
@2002Bishwajeet Great work. Just small change 🚀

File: composer.lock
Comment (christyjacob4) [inline composer.lock:44]: I dont think this file needs to be a part of this change

Comment (2002Bishwajeet) [reply composer.lock:44]: Yup, that's why I removed this file. It's autogenerated right?

Comment (2002Bishwajeet) [reply composer.lock:44]: I was trying to make a custom template for C# so that time it was built

File: .github/ISSUE_TEMPLATE/bug.yaml
Comment (christyjacob4) [inline .github/ISSUE_TEMPLATE/bug.yaml:37]: @2002Bishwajeet  We need to replace this with `Appwrite Version` and add all the currently active Appwrite versions  as dropdown options

**Review by christyjacob4 — CHANGES_REQUESTED** [review]

**Review by christyjacob4 — APPROVED** [review]

**Comment by christyjacob4** [comment]
THANK YOU! All changes merged 🥳

Please reach out to me on our [Discord server](https://appwrite.io/discord) if you would like to claim your Appwrite swags! As a way of saying thank you, we would also love to invite you to join the Appwrite organization on GitHub. Please share your GitHub username with us on Discord.  

You can accept the invite by visiting https://github.com/orgs/appwrite/invitation. By joining our team, you will officially be an Appwrite maintainer on GitHub.

You can change your membership [visibility settings](https://help.github.com/en/github/setting-up-and-managing-your-github-user-account/publicizing-or-hiding-organization-membership), so your new Appwrite team membership badge will show up on your personal GitHub profile.

Please feel free to look for more PRs you might be interested in helping with on our long list of [Hacktoberfest friendly issues](https://github.com/search?q=org%3Aappwrite+org%3Autopia-php+is%3Aissue+label%3Ahacktoberfest+created%3A%3E2021-01-01+is%3Aopen&type=issues) and help make Appwrite better :)

**Pull request merged**
